src/pybind/mgr/dashboard/run-tox.sh

```diff
@@ -24,4 +24,4 @@ if [ "$WITH_PYTHON3" = "ON" ]; then
   ENV_LIST+="py3-cov,py3-lint"
 fi
 
-tox -c ${TOX_PATH} -e $ENV_LIST
+CEPH_BUILD_DIR=$CEPH_BUILD_DIR tox -c ${TOX_PATH} -e $ENV_LIST
```

Comment:
probably we could change line 16~18 to `export CEPH_BUILD_DIR`.

Reply:
done